corgi/core/files.py

```diff
@@ -76,6 +77,11 @@ def render_content(self) -> str:
             "released_components": released_components,
             "distinct_provides": distinct_provides,
         }
+
+        if fixup:
+            # apply temporary cpe lookup fixup
+            kwargs_for_template["cpes"] = cpe_lookup(self.obj.name)  # type: ignore
```

Comment:
Let's pull out the CPEs into a "cpes" kwarg depending on the flag value instead. We want one set of CPEs or the other, not both. So:

```
        kwargs_for_template = {
            "obj": self.obj,
            "released_components": released_components,
            "distinct_provides": distinct_provides,
            "cpes": cpes.get(self.obj.name) if use_cpe_mapping else self.obj.cpes,
        }
```

This way you also don't have to modify the template and only change `obj.cpes` to `cpes`.